src/components/home.js

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { FaSearch } from 'react-icons/fa';
+import { fetchCoins } from '../redux/currency/currency';
+import Display from './display';
+import './home.css';
+
+const Home = () => {
+  const dispatch = useDispatch();
+
+  const { coins, status } = useSelector((state) => state.coins);
+  const [search, setSearch] = useState('');
+  useEffect(() => {
+    if (status === null) {
+      dispatch(fetchCoins());
+    }
+  }, [status, dispatch]);
+
+  return (
+    <div>
+      <div className="imgDv">
+        <img className="main_image" src="https://images.pexels.com/photos/8919542/pexels-photo-8919542.jpeg?auto=compress&cs=tinysrgb&w=600" alt="" />
+      </div>
+      <div className="searchDiv">
+        <input
+          className="searchBar"
+          type="text"
+          value={search}
+          onChange={(e) => setSearch(e.target.value)}
+          placeholder="Search Here"
+        />
+        <FaSearch className="searchIcon" />
+      </div>
+      <div className="grid">
+        {coins.filter((searchCoin) => searchCoin.name.toLowerCase()
+          .includes(search.toLowerCase())
+              || searchCoin.symbol.toLowerCase().includes(search.toLowerCase()))
+          .map((coins) => (
+            <div className="item" key={coins.id}>
+              <NavLink className="coinLink" state={coins} to="/display">
```

Comment:
This is part of the project requirements to have a unique link for each item of your page, for example, you can have `https://crypto-jonmun.netlify.app/{bitcoin}` instead of `https://crypto-jonmun.netlify.app/{display}`, please make sure to have a unique route for each items.